### PR TITLE
Update for 2.4.5 release

### PR DESCRIPTION
**Pull request by dtrudg**
Hi @vsoch - I think this is all that's needed to have the site directing people to the new 2.4.5

Would you be able to review these for an update to the site? Thanks!

### CODE REVIEW

**Comment by vsoch** [comment]
yeah! Let me give it a quick look.

**Comment by vsoch** [comment]
The links to files look good - let's add links to both @jtriley and the two PRs mentioned so the user can follow easily to learn more. You should be able to use {{ site.repo }} to refer to the start of the github links, in case they change in the future.

**Review by vsoch — CHANGES_REQUESTED** [review]
- [ ] add link to @jtriley reference
- [ ] add links to two PRs requested using {{ site.repo }}

**Comment by vsoch** [comment]
And congrats on the release! :muscle:

**Comment by vsoch** [comment]
**Q**: Why was @dctrud excited to go to the market?
**A**: They were having a deal on ice cream! It was 2.4.5 :)

**Comment by dtrudg** [comment]
@vsoch - I think ^^ should do it.

**Comment by vsoch** [comment]
ok taking a look! The text looks spot on, just want to make sure!

**Comment by vsoch** [comment]
ah one more detail, and this is my fault because I just forgot about it (and one of my biggest pet peezes with websites!). The link icon that is being used says "this opens externally" but the links here don't, so you would want to add a class "no-after" to them, like this:

```
<a href="https://github.com/singularityware/singularity/pull/1387" class="no-after">1387</a>/<a class="no-after" href="https://github.com/singularityware/singularity/singularity/pull/1397">1397</a>
```
and for jtriley

```
<a href="https://github.com/jtriley" class="no-after">@jtriley</a>
```
OR if you want to keep the external link icon (and make it an external link, which is also a good idea) just add the target attribute to be _blank:

```
<a href="https://github.com/jtriley" target="_blank">@jtriley</a>
```

**Comment by dtrudg** [comment]
Hi @vsoch - I'm kinda confused about these link class requests. I wrote the new links in markdown, so I don't have the raw html to insert a class into, and I think the icons are correct for the links.

An external link is one on a different site/domain. not something that has target _blank.

> External Links are hyperlinks that point at (target) any domain other than the domain the link exists on (source).
>
>In layman's terms, if another website links to you, this is considered an external link to your site. Similarly, if you link out to another website, this is also considered an external link.

https://moz.com/learn/seo/external-link

These links are at GitHub, and we are lbl.gov so it's correct that they are external links. The '_blank' target isn't an external link... it's just to open a link (external or not) in a new window or tab.

**Comment by vsoch** [comment]
It's a link that opens in a new window. The html can render in the markdown file okay, give it a try! And the class already exists - I made it for this exact purpose :)

**Comment by dtrudg** [comment]
I completely disagree here - the definition of an external link is **not** one that opens in a new window. - it **is** one that links to a different domain. There has been convention on some sites to open external links in a new window, but that's **not** related to the distinction between an external or internal link. 

Also - it is really weird to write raw HTML in markdown for links when there is markup for it.

I will make the change though to have the links target _blank though in the interests of getting this doc up ASAP.

**Review by vsoch — COMMENTED** [review]
Sorry I misspoke, I meant external tab! I don't like link that open a new windows either. It is frustrating however when you are reading a site and you open something that you think will open a new tab and you completely lose the page that you were reading. That's more along the lines that I'm thinking of here!

File: _posts/releases/2018-03-19-release-2-4-5.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+title:  "Singularity 2.4.5 Release"
+category: releases
+permalink: "release-2-4-5"
+version: "2.4.5"
+---
+
+This is a security-related point release, bringing the following fix thanks to Justin Riley ([@jtriley](https://github.com/jtriley)):
+
+PR<a target="_blank" href="{{ site.repo }}/pull/1387">1387</a>/<a target="_blank" href="({{ site.repo }}/singularity/pull/1397">1397</a> - python: strip "Authorization" header on (urllib) redirects to different domains
+
```
Comment (vsoch) [inline _posts/releases/2018-03-19-release-2-4-5.md:11]: This second link might render to two Singularity?

**Comment by dtrudg** [comment]
Ok, cool - preference noted. I'm a 'more than 5 tabs open is oooof' kinda guy, and tend to dislike links targetting new windows unless I ask for it with Ctrl.

Fingers crossed the html is good now, so we can get the 2.4.5 reference up in the docs, since it's a security release. Cheers!

**Review by vsoch — COMMENTED** [review]
hey there is still a small typo - I know this is important and you already responded so many times! Let me pull it and I'll fix it up and get it merged asap :) I hope you went to sleep!

**Comment by vsoch** [comment]
Looks great @dctrud ! Thank you! http://singularity.lbl.gov/release-2-4-5

**Comment by vsoch** [comment]
oh and uh, if you ever see my computer, maybe don't look too closely at the number of tabs... 
image :)
